frontend/src/scenes/billing/PurchaseCreditsModal.tsx

```diff
@@ -0,0 +1,202 @@
+                                        </>
+                                    ) : (
+                                        '25% off'
+                                    ),
+                                prefix: '$',
+                                value: 60000,
+                                top: true,
+                            },
+                            {
+                                type: BillingGaugeItemKind.FreeTier,
+                                text:
+                                    +creditForm.creditInput >= 100000 ? (
+                                        <>
+                                            <IconCheckCircle className="text-success" /> 30% off
+                                        </>
+                                    ) : (
+                                        '30% off'
+                                    ),
+                                prefix: '$',
+                                value: 100000,
+                                top: true,
+                            },
+                            {
+                                type: BillingGaugeItemKind.CurrentUsage,
+                                text: 'Credits purchased',
+                                prefix: '$',
+                                value: +creditForm.creditInput,
+                                top: false,
+                            },
+                        ]}
+                        // @ts-expect-error
+                        product={{
+                            percentage_usage: 0.3,
+                        }}
+                    />
+
+                    <LemonTable
+                        showHeader={false}
+                        columns={[
+                            {
+                                title: '',
+                                dataIndex: 'item',
+                            },
+                            {
+                                title: '',
+                                dataIndex: 'value',
+                            },
+                        ]}
+                        dataSource={[
+                            {
+                                item: "Total credits you'll receive",
+                                value: `$${(+creditForm.creditInput / (1 - creditDiscount)).toLocaleString('en-US', {
+                                    minimumFractionDigits: 0,
+                                    maximumFractionDigits: 0,
+                                })}`,
+                            },
+                            {
+                                item: 'Discount',
+                                value: `${creditDiscount * 100}%`,
+                            },
+                            {
+                                item: 'Due today',
+                                value: `$${Math.round(+creditForm.creditInput).toLocaleString('en-US')}`,
+                            },
```

Comment:
I'd prob put the "Monthly distribution" here so people know that they'll be receiving a chunk of these monthly instead of all at once. Maybe info text: "Credits are dispersed monthly and roll over to the next month. If you use more than the available credits in any month, you'll pay for the usage at the standard rate. Credits expire after 1 year from purchase."